blocks/dashboard/renderAdmin.js

```diff
@@ -1,15 +1,134 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable no-undef */
 import {
   OOPS,
   SCRIPT_API,
   waitForAuthenticated,
   loadingSpinner,
+  dateToRelativeString,
 } from '../../scripts/scripts.js';
 import { readQueryParams, removeQueryParams, writeQueryParams } from '../../libs/queryParams/queryParams.js';
 import paginator from '../../libs/pagination/pagination.js';
 import renderAnalytics from '../../scripts/analytics.js';
+import { toClassName } from '../../scripts/aem.js';
 
+const langNames = new Intl.DisplayNames(['en'], { type: 'language' });
+function parseAcceptLanguage(str) {
+  if (!str) return null;
+  return langNames.of(str.split(',')[0]);
+}
+
+// header: sec-ch-ua
+function parseBrowser(str) {
+  if (!str) return null;
+
+  const browserParts = str.split(',');
+  try {
+    let output = '';
+    for (const browserStr of browserParts) {
+      const browserName = browserStr.split(';')[0].trim().replaceAll('"', '');
+
+      if (/not\?a_brand/i.test(browserName)) continue;
+      if (/chromium/i.test(browserName)) {
```

Comment:
Other browsers are not captured I guess eg Firefox etc ?

Reply:
Looks like firefox doesn't send that info anyway. I just tried